pwa/src/components/translations/table.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from "react";
+// import "../../style/table.css";
```

Comment:
You missed a comment (I don't think the file exists, so this comment should just be removed).